pkg/network/ebpf/c/protocols/classification/maps.h

```diff
@@ -7,8 +7,7 @@
 #include "protocols/classification/structs.h"
 
 // Kernels before 4.7 do not know about per-cpu array maps.
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0)
-
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0) || defined(COMPILE_CORE)
```

Comment:
if we have CORE, it means that we have percpu array?

Reply:
Nope, but we can't check the kernel version on co-re (yet). The fentry tracer is the only one using co-re at the moment and that is only available in kernel 5.5+. We will need to revisit this once we have the kprobe co-re tracer.